pandas/core/groupby/groupby.py

```diff
@@ -2491,6 +2491,17 @@ def ohlc(self) -> DataFrame:
     @doc(DataFrame.describe)
     def describe(self, **kwargs):
         with self._group_selection_context():
+            if len(self._selected_obj) == 0:
+                if self._selected_obj.ndim == 1:
+                    return self._selected_obj.describe(**kwargs).to_frame().T.iloc[:0]
```

Comment:
can you combine these expressions a bit e.g.
```
described = self._selected_obj.describe(...)
if self._selected_obj.ndim= 1:
    result = described  
else:
    result = described.unstack()
return result.to_frame().T.iloc[:0]
```

Reply:
Have updated